lib/jormungandr/test/integration/Cardano/Faucet.hs

```diff
@@ -0,0 +1,147 @@
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE TypeApplications #-}
+
+module Cardano.Faucet
+    ( initFaucet
+    ) where
+
+import Prelude
+
+import Cardano.Wallet.Primitive.Mnemonic
+    ( mkMnemonic )
+import Control.Concurrent.MVar
+    ( newMVar )
+import Data.Text
+    ( Text )
+import Test.Integration.Faucet
+    ( Faucet (..) )
+
+
+-- | Initialize a bunch of faucet wallets and make them available for the
+-- integration tests scenarios.
+initFaucet :: IO Faucet
+initFaucet = do
+    let wallets =
+            map
+            ((either (error "cannot retrieve mnemonics") id . mkMnemonic @15) . fst)
+            mnemonicsAdresses
```

Comment:
minor style suggestion, it reads better to define an intermediate function for this kind of things, like 

`let wallets = unsafeMkMnemonic @15 . fst <$> mnemonicsAddresses`